decidim-core/app/controllers/decidim/profiles_controller.rb

```diff
@@ -83,7 +83,7 @@ def ensure_profile_holder
     def profile_holder
       return if params[:nickname].blank?
 
-      @profile_holder ||= Decidim::UserBaseEntity.find_by(nickname: params[:nickname].downcase, organization: current_organization)
+      @profile_holder ||= Decidim::UserBaseEntity.find_by("LOWER(nickname) = ? AND decidim_organization_id = ?", params[:nickname].downcase, current_organization.id)
```

Comment:
Here isn't it redundant to lower the nickname with SQL and ruby ?

```suggestion
      @profile_holder ||= Decidim::UserBaseEntity.find_by(nickname: params[:nickname].downcase, organization: current_organization)
```

Reply:
It's useful to compare downcased nicknames in db from the downcased nickname in params